src/util.ts

```diff
@@ -370,32 +372,27 @@ export class Util {
     }
 
     /**
-     * Given an absolute path to a source file, and a target path,
-     * compute the pkg path for the target relative to the source file's location
-     * @param containingFilePathAbsolute
-     * @param targetPath
+     * Compute the pkg path for the target relative to the source file's location
+     * @param sourcePkgPath The pkgPath of the file that contains the target path
+     * @param targetPath a full pkgPath, or a path relative to the containing file
      */
-    public getPkgPathFromTarget(containingFilePathAbsolute: string, targetPath: string) {
-        //if the target starts with 'pkg:', it's an absolute path. Return as is
-        if (targetPath.startsWith('pkg:/')) {
-            targetPath = targetPath.substring(5);
-            if (targetPath === '') {
-                return null;
-            } else {
-                return path.normalize(targetPath);
-            }
-        }
-        if (targetPath === 'pkg:') {
+    public getPkgPathFromTarget(sourcePkgPath: string, targetPath: string) {
+        //handle some edge cases
+        if (targetPath === 'pkg:' || targetPath === 'pkg:/') {
             return null;
         }
+        //if the target starts with 'pkg:', return as-is
+        if (targetPath.startsWith('pkg:/')) {
+            return targetPath;
+        }
 
         //remove the filename
-        let containingFolder = path.normalize(path.dirname(containingFilePathAbsolute));
+        let containingFolder = path.posix.normalize(path.dirname(sourcePkgPath));
```

Comment:
It looks like `path.posix.normalize` does NOT replace `\` with `/`

Reply:
Oh, yeah good call. I didn't realize that. I still want to use normalize so we can fix any `../` or double slashes, but I refactored this to use a regex split on either path separator just to be safe.

only pkgPaths should be passed to this, but better safe than sorry i guess?